site/content/en/docs/user/resource-usage-configuration.md

```diff
@@ -0,0 +1,134 @@
+---
```

Comment:
This page is actually similar to Exec, Logs, Attach, PortForward. Let's make the content and style of the descriptions consistent across multiple pages.